db/migrate/20160427110426_create_services.rb

```diff
@@ -0,0 +1,10 @@
+class CreateServices < ActiveRecord::Migration
+  def change
+    create_table :services do |t|
+      t.integer :identifier
```

Comment:
The ESD datasets call this number the identifier, but we've been referring to it between ourselves as the LGSL code (and LGIL code for the interaction one). It would be good to use the name that our users call it - maybe Jane can help us with that question.